### PR TITLE
Task s3 nav

### PR DESCRIPTION
**Pull request by nsantiagoblair**

### CODE REVIEW

File: css/splash.css [hunk 1]
```diff
@@ -24,7 +27,6 @@
 
 .splash .instructionsBtn:hover {
    cursor: pointer;
-   background-color: #dddddd;
    text-shadow: 2px 2px 2px #000000, 2px -2px 2px #000000, -2px -2px 2px #000000, -2px 2px 2px #000000;
    box-shadow: 1px 1px 2px #000000, 1px -1px 2px #000000, -1px -1px 2px #000000, -1px 1px 2px #000000;
 }
```
Comment (C0deJack) [inline css/splash.css:32]: can we reduce this code?

File: css/styles.css [hunk 2]
```diff
@@ -11,6 +11,5 @@ body {
     height: 800px;
     margin: 0 auto;
     border: 1px solid black;
-    background: rgb(63,94,251);
-    background: radial-gradient(circle, rgba(63,94,251,1) 0%, rgba(252,70,132,1) 100%);
+
```
Comment (C0deJack) [inline css/styles.css:14]: Is this separate CSS file required?

File: includes/loadTrack.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+
```
Comment (C0deJack) [inline includes/loadTrack.js]: blank line at top . Also this file should be in the JS folder

File: includes/loadTrack.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    } else if (link === '2' && $('#track2').length !== 0) {
+        $('.track').css('display', 'none')
+        $('#track2').css('display', 'block')
+    }
+
+    $('#splash').css('display', 'none')  
+})
+
+$('body').on('click', '.trackReturn', function(e) {
+    $('.track').css('display', 'none')
```
Comment (C0deJack) [inline includes/loadTrack.js]: there is an 'e' thats unused

**Review by C0deJack — CHANGES_REQUESTED** [review]
Instructions button needs fixing

File: css/splash.css (hunk 1 above)
Comment (nsantiagoblair) [reply css/splash.css:32]: No.

File: css/styles.css (hunk 2 above)
Comment (nsantiagoblair) [reply css/styles.css:14]: It doesn't pertain to this task

File: includes/loadTrack.js (hunk 3 above)
Comment (nsantiagoblair) [reply includes/loadTrack.js]: 🎉

File: includes/loadTrack.js (hunk 4 above)
Comment (nsantiagoblair) [reply includes/loadTrack.js]: 🎉

**Review by C0deJack — APPROVED** [review]
All good

**Pull request merged**